inst/rmarkdown/templates/ims/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,422 @@
+---
+journal: aoas 
+title: A sample article with some additional note
+thanks: # delete thanks field if not require, here notes for title
+  - ref: T1 
+    text: A sample additional note to the title.
+runtitle: A sample running head title
+author:
+  - name: First 
+    surname: Author
+    email: \email{first@somewhere.com}
+    label: e1
+    addressLabel: A
+    sepNext: "," # use "," or \and; the last author not need this field
+  - name: Second
+    surname: Author
+    email: \email{second@somewhere.com}
+    label: e2,mark
+    addressLabel: B
+    sepNext: \and
+  - name: Third
+    surname: Author
+    email: \email{third@somewhere.com}
+    label: e3,mark
+    addressLabel: B
+affiliation:
+  - label: A
+    name: Department, University or Company Name
+    authorsLabels: e1
+  - label: B
+    name: Department, University or Company Name
+    authorsLabels: e2,e3
+abstract: |
+  The abstract should summarize the contents of the paper.
+  It should be clear, descriptive, self-explanatory and not longer
+  than 200 words. It should also be suitable for publication in
+  abstracting services. Formulas should be used as sparingly as
+  possible within the abstract. The abstract should not make
+  reference to results, bibliography or formulas in the body
+  of the paper---it should be self-contained.
+
+  This is a sample input file.  Comparing it with the output it
+  generates can show you how to produce a simple document of
+  your own.
+keywords: 
+  - First keyword
+  - Second keyword
+  
+bibliography: ims.bib
+biblio-style: imsart-nameyear # imsart-number
+citation_package: natbib
```

Comment:
The template is supposed to work only with `natbib`. In accordance, I removed the `citation_package` field in the YAML. 

On the other hand, I couldn't make that Rmarkdown generates the cited references with a hyperlink to the references section (as in the journal example).  Also, as far as I know, the Rmarkdown parsing would generate the citations using `\citep` and similars, oppose to the `\citeauthoryear` used in the tex template by the journal. I'm afraid this could be a problem with the journal. 

For now, I'm using in the skeleton the commands recommended in the journal template. I also added the content for Pandoc citeproc in the template, but It's commented bc the mention reasons above. Everything would work uncommenting those lines, but it won't produce the expected output. 

I'm happy to read any suggestion about, maybe from other template experiences.

Reply:
If this template is supposed to work only with `natbib` then, the function `ims_article` should have that by default for `citation_package`

```r
ims_article <- function(..., keep_tex = TRUE, citation_package = "natbib") {
  pdf_document_format(
    "ims", keep_tex = keep_tex, citation_package = citation_package, ...
  )
}
```

Does it makes sense ? Otherwise, natbib won't be used by Pandoc. Currently, it is Pandoc citeproc which is used, not natbib directly.

Reply:
Thanks for the clarification. I changed the function and template accordingly.